src/services/post.service.ts

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
```

Comment:
This service file is importing React but it's not being used anywhere in the service. This import can be removed.